hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/v2/MiniMRYarnCluster.java

```diff
@@ -259,7 +267,10 @@ public void run() {
           };
         }.start();
```

Comment:
historyServer is not initialized if `mapreduce.jobhistory.minicluster.enabled` is set to `false`. I would like to keep the default value to be `true` to avoid breaking existing tests of hadoop and downstreams.

Reply:
Yes, preserving the default value makes sense. I'm just wondering if the resource allocation can be skipped entirely when the `jhs` is disabled.